src/components/ChatEntry.js

```diff
@@ -2,21 +2,66 @@ import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
+const convertMsToDays = (ms) => {
```

Comment:
Nice job writing the time handling yourself.  🙂

(There is a library you can use for it, but it's educational to implement something like this yourself.)